controllers/cloud.redhat.com/providers/autoscaler/default.go

```diff
@@ -20,6 +20,10 @@ func NewAutoScalerProvider(p *p.Provider) (p.ClowderProvider, error) {
 }
 
 func (asp *autoscalerProvider) Provide(app *crd.ClowdApp, c *config.AppConfig) error {
+	if !app.PreHookDone() {
```

Comment:
We need to be very careful about skipping here, as skipping parts of providers can mean the cache is not furnished with information for subsequent providers. I'm not quite sure of the solution yet, but this is a little scary.

Reply:
Oh - And we definitely need to log something here